.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
```

Comment:
Add this to the gitignore so we don't commit IDE config files

Reply:
Added to the gitignore and removed from the repo